sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,53 @@
+# Basic device update enumerations
```

Comment:
The samples do not comply our guideline. Please refer to https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples and https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/Create_Samples_Guidance.md to update the samples. and You can refer to the SDK package template https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template/Azure.Template

Reply:
Can we ship this preview with these samples and address the feedback in our next PR?

Reply:
As we discussed in the email, that for preview, we can ship this without compliance samples.